scaaml/plot.py

```diff
@@ -70,13 +71,13 @@ def plot_comparaison(traces, labels, title):
     for idx, trace in enumerate(traces):
         label = labels[idx]
         if label == "SNR":
-            color = '#8E24AA'
-        elif 'activation' in label.lower():
-            color = '#64DD17'
-        elif 'grad' in label.lower():
-            color = '#26A69A'
-        elif 'scald' in label.lower():
-            color = '#03A9F4'
+            color = "#8E24AA"
```

Comment:
I wonder how hard this would be to use a lookup `dict` here instead.
But at least we could factor out `label.lower`:
```python
label = labels[idx].lower()
if label == "snr":
...
```